src/types/server_decoration.zig

```diff
@@ -0,0 +1,52 @@
+const wlr = @import("../wlroots.zig");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+
+pub const ServerDecorationManager = extern struct {
+    pub const Mode = enum(c_int) {
+        none = 0,
+        client = 1,
+        server = 2,
+    };
+
+    global: *wl.Global,
+    resources: wl.list.Head(wl.Resource, null),
+    decorations: wl.list.Head(ServerDecoration, "link"),
+
+    default_mode: Mode,
```

Comment:
This is a `uint32_t` which does not have the same ABI as the enum on all targets.